com.unity.render-pipelines.high-definition/Documentation~/HDR-Output.md

```diff
@@ -0,0 +1,96 @@
+# High Definition Range (HDR) Output in HDRP
+
+With HDRP it is possible to output to High Dynamic Range (HDR) devices that allows for wider color gamut and higher brightness range.
+
+HDR Output can be enabled for a given platform in **Player Settings > Other Settings > Use display in HDR mode**.  When this option is enabled, HDRP will process the frame so that when an HDR screen is used as output device it is presented correctly.
+
+## Tonemapping for HDR Output
+
+When the *Use display in HDR mode* option is set in the player settings, all the HDR related options appear in the  [Tonemapping](Post-Processing-Tonemapping.md) volume component. The available options depend on the Tonemapping mode selected.
```

Comment:
Proposed rewrite:
"Enable **Use display in HDR mode**  to reveal HDR-related options in the  [Tonemapping](Post-Processing-Tonemapping.md) [Volume](https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@13.0/manual/Volumes.html) component. 

Each **Tonemapping** mode has a unique set of properties. "

Reply:
> Each Tonemapping mode has a unique set of properties

This is a bit confusing to me as many properties are shared, I am adding it as 

> Each **[Tonemapping](Post-Processing-Tonemapping.md)** mode might have different available set of properties. 

Let me know if sounds good to you

Reply:
Ok, I see this change.